projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -402,28 +402,31 @@ const VideoPressEdit = CoreVideoEdit =>
 				</Fragment>
 			);
 
-			if ( isUploading ) {
-				return (
-					<Fragment>
-						{ blockSettings }
-						<Loading text={ __( 'Uploading…', 'jetpack' ) } />
-					</Fragment>
-				);
-			}
+			const isFetching = ! isUploading && ( isFetchingMedia || isFetchingPreview );
```

Comment:
I'm wondering if this can be simplified to just the 2 `fetching` variables, since `isUploading` is always checked when `isFetching` is in a condition in the new code below.

Reply:
✅